sandbox/git-annex-remote-gitobjects

```diff
@@ -37,7 +37,7 @@ setconfig () {
 
 # Sets ref to the reference to use to store a key.
 calcref () {
-	ref="refs/annexed/$1"
+	ref="refs/gitobjects/$1"
```

Comment:
I see the value of using the name of the special remote, but I think it'd still for good for there to be "annex" in the name.  Length shouldn't matter here, so what about "refs/annex-gitobjects" or "refs/gitobjects-annexed"?

Reply:
Yes, I thought about that too but went for shorter for no good reason. Probably refs/annex-gitobjects would be best